v2/features/dill/RevenueStats.tsx

```diff
@@ -76,6 +81,21 @@ const RevenueStats: FC<Props> = ({ dill }) => {
         <div className="mb-6 xl:mb-0">
           <h2 className="font-title font-medium text-foreground text-lg leading-5">
             {formatPercentage(dillAPY)}
+            <MoreInfo>
+              <p className="font-bold text-primary">{`${t("Rewards")}`}</p>
+              {
+                <div className="flex justify-between items-end">
+                  <div className="font-bold text-foreground text-xs mr-2">ETH:</div>
+                  <div className="text-foreground text-xs">{formatPercentage(ethRewards)}</div>
+                </div>
+              }
+              {
+                <div className="flex justify-between items-end">
+                  <div className="font-bold text-foreground text-xs mr-2">PICKLE:</div>
+                  <div className="text-foreground text-xs">{formatPercentage(pickleRewards)}</div>
+                </div>
```

Comment:
<img width="173" alt="Screen Shot 2022-05-18 at 21 00 48" src="https://user-images.githubusercontent.com/7811733/169059764-46067d3e-d0a8-4b10-8d35-bca735246da7.png">

PICKLE shows up as 0

Reply:
This is just waiting for the latest pf-core release. It should come up once that is done.